packages/twenty-front/src/modules/views/view-picker/hooks/useViewPickerPersistView.ts

```diff
@@ -20,9 +21,10 @@ export const useViewPickerPersistView = () => {
 
   const { createView, selectView, removeView, updateView } = useHandleViews();
 
-  const { viewsOnCurrentObject } = useGetCurrentView();
+  const { viewsOnCurrentObject, componentId } = useGetCurrentView();
```

Comment:
componentId shouldn't be used like that if what we intend is to use objectNamePlural, doesn't matter if componentId === objectNamePlural

Reply:
You can even modify useGetCurrentView to return objectNamePlural